src/test/java/io/stargate/sgv2/jsonapi/api/v1/DeleteOneIntegrationTest.java

```diff
@@ -148,12 +148,10 @@ public void noOptionsAllowed() {
           .body("data.document", is(nullValue()))
           .body("status", is(nullValue()))
           .body("errors", is(notNullValue()))
-          .body("errors", hasSize(2))
+          .body("errors", hasSize(1))
           .body("errors[0].message", is("Command accepts no options: DeleteOneCommand"))
-          .body("errors[0].exceptionClass", is("JsonMappingException"))
```

Comment:
Due to interaction or various catch blocks, had `JsonApiException` "hidden" inside `JsonMappingException` (wrong way around). But we also do not necessarily want to expose internal/intermediate `JsonMappingException` at all.